functions.php

```diff
@@ -0,0 +1,26 @@
+<?php
+function getDatabase(): PDO{
+    $db = new PDO('mysql:host=DB;dbname=collectorApp', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function queryDatabase(PDO $db){
```

Comment:
not a very good function name. qhat does the query do? maybe try `getCharacters`

Reply:
also missing your return type hint

Reply:
Updated to getCharacters, I see what you mean. Also added array as return type.